modules/core/src/materials/phong-material.js

```diff
@@ -9,7 +9,7 @@ const defaultProps = {
 
 export default class PhongMaterial extends Material {
   constructor(props) {
-    super(props);
+    super();
```

Comment:
Hmm... This looks a bit weird. Why not forward the props? I think we will need to restore this once we put some props in the base class.